### ISSUE TITLE
[nuxt] useRuntimeConfig not defined on 1.1.0

### ISSUE THREAD

**Issue by hi-reeve** [prazdevs/pinia-plugin-persistedstate#181]
### Describe the bug

after update to `pinia-plugin-persistedstate/nuxt` version `1.1.0` i got the error `useRuntimeConfig is not defined`

### Reproduction

-

### System Info

```Shell
System:
    OS: macOS 13.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 241.50 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.12.0 - ~/.nvm/versions/node/v18.12.0/bin/node
    npm: 8.19.2 - ~/.nvm/versions/node/v18.12.0/bin/npm
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by leggod** [comment]
Me, too. It's so annoying.

**Comment by leggod** [comment]
How to put the dependency, is it like me?

```
"devDependencies": {
    "@pinia-plugin-persistedstate/nuxt": "^1.1.0",
    "@vant/nuxt": "^1.0.0",
    "less": "^4.1.3",
    "less-loader": "^11.1.0",
    "nuxt": "^3.2.3",
    "postcss-px-to-viewport-8-plugin": "^1.2.0"
  },
  "dependencies": {
    "@pinia/nuxt": "^0.4.7",
    "pinia": "^2.0.33",
    "vue-horizontal": "^1.0.2"
  }
```

**Comment by prazdevs** [comment]
👋 should be fixed with @1.1.1
